tools/depends/pre-depends/m4-pre-depends/Makefile

```diff
@@ -1,19 +1,20 @@
 include ../../Makefile.include.in
```

Comment:
> ../../Makefile.include doesn't exist at the time this file is used

Yes, best would be to separate the configured and the unconfigured (static) stuff into two files. Just nitpicking though ;-)